apheleia-formatters.el

```diff
@@ -269,6 +272,7 @@ rather than using this system."
     (LaTeX-mode . latexindent)
     (lua-mode . stylua)
     (lisp-mode . lisp-indent)
+    (markdown-mode . dprint)
```

Comment:
I've been holding off on installing a default formatter for markdown because people have so many different ways of formatting it across different projects. Will add a comment noting as such so it's clear why there is seemingly an entry missing.

Reply:
Ok, good, removed in 7074bec0c0884a23a7ba959b61026ff18432b069.

(I'm assuming you'll eventually squash this PR on merge, so keeping the branch history here more reviewable than tidy.)